test/e2e/cluster_extension_install_test.go

```diff
@@ -224,12 +223,12 @@ func TestClusterExtensionBlockInstallNonSuccessorVersion(t *testing.T) {
 		}
 		assert.Equal(ct, ocv1alpha1.ReasonResolutionFailed, cond.Reason)
 		assert.Equal(ct, "error upgrading from currently installed version \"1.0.0\": no package \"prometheus\" matching version \"1.2.0\" found", cond.Message)
-		assert.Empty(ct, clusterExtension.Status.ResolvedBundle)
+		// TODO (dfranz) do we still need to be clearing this status field when this error is reached?
```

Comment:
Another `TODO` here. The `ResolvedBundle` status is not being cleared; I'm not sure if it was intentional or if it just got dropped in a rebase. It's easy to add back in, though.

Reply:
We can clear the `ResolvedBundle` status for now, to make sure there is no functionality being changed.

But there is a nuance here with the upgrade case, where we would probably not want to clear `Installed` during upgrade (though clearing resolved seems right). But this nuance will be handled in #842 - where we can discuss more on status conditions.

Reply:
Added this check back in and clearing `ResolvedBundle` field when we call `handleResolutionErrors`.